extensions/wikia/ArticleVideo/bower_components/skin-config/validate.js

```diff
@@ -0,0 +1,71 @@
+var fs = require( 'fs' );
+var stripComments = require( 'strip-json-comments' )
+
+// assumes this is run via node.
+function usage( message ) {
+    if( message ) {
+        console.log( message );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)